client/src/main/java/de/rebleyama/client/RebleyamaClient.java

```diff
@@ -55,8 +65,10 @@ public void render() {
         camera.update();
 		
 		//start up map renderer
```

Comment:
Indentation of this comment looks weird. Please use consistent code formatting.

Reply:
Fixed. Additional newline from merge conflict was the issue